agent/lib_monolog.c

```diff
@@ -432,6 +493,47 @@ NR_PHP_WRAPPER(nr_monolog_logger_pushhandler) {
                 "application may be sending logs to New Relic twice.");
   }
 
+  if (nr_txn_log_decorating_enabled(NRPRG(txn))) {
+    zval* callback_name = NULL;
+    zval* ph_retval = NULL;
```

Comment:
What does the `ph_` prefix in `ph_retval` stand for?

Reply:
(P)ush(H)andler

Reply:
Ah! The method name, however, is `pushProcessor` and that's what got me thinking: 'Why `ph_`?' No need to change it - it's a local variable in a limited scope so the name is not that important. I was just curious what was the history behind `ph_`.